.github/workflows/swift-toolchain.yml

```diff
@@ -2118,9 +2118,119 @@ jobs:
           symbolsFolder: ${{ github.workspace }}/BinaryCache
           searchPattern: '**/*.exe'
 
+  debugging_tools:
+    name: Debugging Tools
+    needs: [context, compilers, devtools, sdk]
+    runs-on: ${{ needs.context.outputs.windows_build_runner }}
+    env:
+      TOOLCHAIN_PATH: ${{ github.workspace }}/BuildRoot/Library/Developer/Toolchains/unknown-Asserts-development.xctoolchain/usr/bin
+      # Where to store new binaries.
+      DEBUGGING_TOOLS_DIR: ${{ github.workspace }}/BuildRoot-DebuggingTools/Library/Developer/Toolchains/unknown-Asserts-development.xctoolchain/usr
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - arch: amd64
+            cpu: x86_64
+            triple: x86_64-unknown-windows-msvc
+
+          # TODO(thebrowsercompany/swift-build#125): Fix the arm64 build and uncomment this.
+          # - arch: arm64
+          #   cpu: aarch64
+          #   triple: aarch64-unknown-windows-msvc
+
+    steps:
+      - name: Download compilers
+        uses: actions/download-artifact@v4
+        with:
+          name: compilers-${{ matrix.arch }}
+          path: ${{ github.workspace }}/BuildRoot/Library
+
+      - name: Download devtools
+        uses: actions/download-artifact@v4
+        with:
+          name: devtools-${{ matrix.arch }}
+          path: ${{ github.workspace }}/BuildRoot/Library
+
+      - name: Download cmark-gfm-amd64
+        uses: actions/download-artifact@v4
+        with:
+          name: cmark-gfm-amd64-0.29.0.gfm.13
+          path: ${{ github.workspace }}/BuildRoot/Library/cmark-gfm-0.29.0.gfm.13/usr
+
+      - name: Download ${{ matrix.arch }} Windows Runtime
+        uses: actions/download-artifact@v4
+        with:
+          name: windows-sdk-${{ matrix.arch }}
+          path: ${{ github.workspace }}/BinaryCache
```

Comment:
Do we not need the runtime for AMD64 always? We have a dependency on it for the toolchain I thought.

Reply:
Oh yes, thanks for catching. This is just a bad copy-paste. I got lucky here that this is currently always evaluating `${{ matrix.arch }}` to `amd64` and didn't notice.